addons/frag/stringtable.xml

```diff
@@ -175,8 +220,40 @@
+            <Spanish>(Solo SP) Seguimiento de depuración de Fragmentación/Astillamiento </Spanish>
+            <Polish>(Tylko SP) Wizualny debug odł./odpr.</Polish>
+            <Czech>(Pouze SP) Debug sledování Frag/Úlomků</Czech>
+            <German>(nur SP) Splitter-/Explosions-Debug-Verfolgung</German>
+            <Portuguese>(Somente SP) Depuração de fragmentação e estilhaços traçantes</Portuguese>
+            <French>(SP uniquement) Fragmentation/éclat debug</French>
+            <Hungarian>(Csak SP) Repesz/Pattogzás debug követés</Hungarian>
+            <Russian>(Только для одиночной игры) Отслеживаение/отладка осколков</Russian>
+            <Italian>(Solo SP) Debug Tracciamento Frag/Spall</Italian>
+            <Japanese>(SP のみ) 破片/剥離のデバッグ用表示</Japanese>
+            <Korean>(싱글플레이 전용) 탄환파편/파편 디버그 추적화</Korean>
+            <Chinesesimp>（仅单人）追踪显示破片粒子</Chinesesimp>
+            <Chinese>(僅在單人模式) 碎片/剝落除錯追蹤</Chinese>
+        </Key>
+        <Key ID="STR_ACE_Frag_EnableDebugTrace_Desc_Old">
+            <English>(SP only) Requires a mission/editor restart. Enables visual tracing of fragmentation and spalling rounds in SP game mode only.</English>
+            <Spanish>(Solo SP) Requiere un reinicio misión/editor. Permite el seguimiento visual de la fragmentación y astillamientos de los proyectiles en modo SP.</Spanish>
+            <Polish>(Tylko SP) Wymaga restartu misji/edytora. Aktywuje wizualne śledzenie odłamków oraz odprysków w trybie gry Single Player.</Polish>
+            <German>(nur SP) Splitter-/Explosions-Debugging</German>
+            <Czech>(Pouze SP) Vyžaduje restart mise/editoru. Aktivuje vizuální stopování fragmentace a úlomů pouze v režimu jednoho hráče.</Czech>
+            <Portuguese>(Somente SP) Requer um reinício de missão / editor. Habilita o rastreamento visual de projéteis de fragmentação e estilhaçamento apenas no modo de jogo SP.</Portuguese>
+            <French>(SP seulement) Requiert un redémarrage de mission ou de l'éditeur. Active les traceurs visuels de fragmentation et d'éclats en mode solo seulement.</French>
+            <Hungarian>(Csak SP) Küldetés/Editor újraindítás szükséges. Engedélyezi a repeszek és pattogzó lövedékek vizuális nyomkövetését, csak egyjátékos módok alatt.</Hungarian>
+            <Russian>(Только для одиночной игры) Требует перезапуска миссии/редактора. Включает визуальные следы от осколков и обломков в режиме одиночной игры.</Russian>
+            <Italian>(Solo SP) Richiede un restart editor/missione. Abilita il tracciamento visivo di schegge da frammentazione/spalling in modalità Giocatore Singolo.</Italian>
+            <Japanese>(SP のみ) ミッションとエディタの再起動が必要です。有効化すると、シングルプレイでのみ破片と剥離の飛翔体が見えるようになります。</Japanese>
+            <Chinesesimp>（仅单人）激活后，只有在单人模式下才可观察到破片粒子的移动轨迹。</Chinesesimp>
+            <Chinese>(僅在單人模式) 讓你在單人模式下可觀察到碎片/剝落粒子的移動軌跡</Chinese>
+            <Korean>(SP 전용) 임무 / 편집자가 다시 시작해야합니다. SP 게임 모드에서만 파편화 및 탄환파편의 시각적 추적을 가능하게 합니다.</Korean>
+        </Key>
```

Comment:
Yeet that:
```suggestion
```

Reply:
Done, apologies on missing that, it was leftover from when I was considering making debug mode fully MP compatible instead of partially

Reply:
Yote https://github.com/acemod/ACE3/pull/9728/commits/fc398ff042718a617e8675336856482149499b53